doc/nrf/libraries/caf/buttons.rst

```diff
@@ -82,11 +82,27 @@ In this state, the module enables the GPIO interrupts and waits for the pin stat
 Whenever a button is pressed, the module switches to ``STATE_SCANNING``.
 When the switch occurs, the module submits a work with a delay set to :kconfig:`CONFIG_CAF_BUTTONS_DEBOUNCE_INTERVAL`.
 The work scans the keyboard matrix, or directly connected buttons (depends on configuration).
-If any button state change occurs, the module sends related event.
+If any button state change occurs, the module sends an event with the :c:member:`button_event.key_id` of that button.
 
 * If the button is kept pressed while the scanning is performed, the work will be resubmitted with a delay set to :kconfig:`CONFIG_CAF_BUTTONS_SCAN_INTERVAL`.
 * If no button is pressed, the module switches back to ``STATE_ACTIVE``.
 
+Key ID
+======
```

Comment:
Please add a blank line after this line.

Reply:
done.